### DISCUSSION TITLE
Error while launching UERANSIM

### DISCUSSION THREAD

**Discussion by Deblina96** [OAI/OpenAPI-Specification#3474] [General]
I am trying to deploy UERANSIM and getting the following error:-
Please explain how this can be solved.
![image](https://github.com/OAI/OpenAPI-Specification/assets/116576465/19a873c0-a35e-4055-b646-5e4fd8670235)

**Comment by lornajane** [comment]
This discussion isn't relevant to this project, please redirect your query to the repository of the project you are using.
